website/src/utils/chat.ts

```diff
@@ -1,19 +1,29 @@
-import { ChatConfigFormData } from "src/types/Chat";
+import { PluginEntry } from "src/types/Chat";
+import { SamplingParameters } from "src/types/Chat";
 
-const CHAT_CONFIG_KEY = "CHAT_CONFIG";
+const CHAT_CONFIG_KEY = "CHAT_CONFIG_V2";
 
-export const setConfigCache = (config: ChatConfigFormData) => {
+export type CachedChatConfig = {
+  model_config_name: string;
+  selectedPresetName: string;
+  custom_preset_config: SamplingParameters;
+  custom_presets: Array<{ name: string; config: SamplingParameters }>;
```

Comment:
This will be used in the next PR when I implement https://github.com/LAION-AI/Open-Assistant/issues/3035